setup.py

```diff
@@ -4,7 +4,7 @@
 from setuptools import setup
 from setuptools.command.test import test as TestCommand
 
-requires = ['six', 'requests', 'datadog']
+requires = ['six', 'requests', 'datadog', 'structlog', 'djangorestframework']
```

Comment:
think we can push DRF into the dev_requires/the docs? We're using panopticon (though obviously not all of it) in some non-django apps.